src/proto/semantifly.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package semantifly;
+
+option go_package = "accretional.com/semantifly/proto";
+
+service Semantifly {
+  rpc Add(AddRequest) returns (AddResponse) {}
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  rpc Get(GetRequest) returns (GetResponse) {}
+  rpc Update(UpdateRequest) returns (UpdateResponse) {}
+  rpc LexicalSearch(LexicalSearchRequest) returns (LexicalSearchResponse) {}
+}
+
+message AddRequest {
+  string index_path = 1;
+  string data_type = 2;
```

Comment:
These should be enums just as they are in the Index proto. In fact you should probably refactor the index proto and this to share message types

Reply:
fixed